packages/zosjobs/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe z/OS jobs SDK package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Fixed error in `DownloadJobs.downloadSpoolContentCommon` causing binary spool files to be corrupted by newline normalization. [#2282](https://github.com/zowe/zowe-cli/issues/2282)
```

Comment:
Can we say what DownloadJob.downblahblah is? As in: "Fixed an error in the `bbbb` function/option/cat treat that corrupted binary spool files by newline normalization."